frontend/src/app/trees/admin/report/report.component.ts

```diff
@@ -150,9 +150,9 @@ export class ReportComponent implements OnInit, AfterViewInit {
       this.setReportParameters();
       this.service.getAllByDateRange(
           this.forest.id,
-          moment.tz(this.form.get('dateTimeRange.startDateTime').value, this.forest.timezone).format('YYYY-MM-DD'),
-          moment.tz(this.form.get('dateTimeRange.endDateTime').value, this.forest.timezone).format('YYYY-MM-DD')
-        )
+          moment.utc(this.form.get('dateTimeRange.startDateTime').value).format('YYYY-MM-DD'),
+          moment.utc(this.form.get('dateTimeRange.endDateTime').value).format('YYYY-MM-DD')
```

Comment:
Is the form being parsed in UTC? I wonder if we can move this into the DateTimeRangeService so that it's easier to see how we're parsing the form and if/what we need to convert in terms of timezones.

Reply:
Since the API only care's about days (YYYY-MM-DD), maybe it makes more sense to just use the individual components (startYear, startMonth, startDay, etc) and avoid timezones entirely on the front end.